src/screens/Home.js

```diff
@@ -145,6 +145,7 @@ const Home = React.memo(function Home() {
           <MultiModal
             visible={createProposalModalVisible}
             onClose={() => setCreateProposalModalVisible(false)}
+            onClosed={() => setCreateProposalModalVisible(false)}
```

Comment:
Is this necessary? Seems that is doing the same as `onClose` which will be fired first?